src/types/model.ts

```diff
@@ -132,12 +146,9 @@ export class Registry {
     genesisBlockNumber: number
   ): Promise<Registry> {
     const db = storage.getDB();
-    const str = await db
-      .get(
-        getNetworkStorageKey(CONDITIONAL_ORDER_REGISTRY_STORAGE_KEY, network)
-      )
-      .then((str) => str)
-      .catch(() => undefined);
+    const ownerOrders = await loadOwnerOrders(storage, network)
+      .then((orders) => orders)
```

Comment:
Is this step needed?
It's not doing anything unless I'm mistaken.

Reply:
I think I'm so used to doing `then` automatically before a `catch`. Always learning hacks with JavaScript :grimacing:

Reply:
That would be avoided if using try/catch instead ;)